.gitattributes

```diff
@@ -1,5 +1,3 @@
-* text=auto
```

Comment:
causes git to use autocrlf=true, which rewrites all newlines to \r\n and cs and ec checks won't pass